test/api/test_catalog.py

```diff
@@ -335,17 +359,19 @@ def assert_metadata_record(schema_id):
 
     if has_datacite:
         assert_metadata_record('SAEON.DataCite4')
+        pass
```

Comment:
I guess this was added to help with debugging, but the `pass` statements are not needed and can be removed.